### ISSUE TITLE
Support for IOS 7

### ISSUE THREAD

**Issue by ussu99** [robertwijas/UISS#29]
Currently compiling for iOS 7 generates warnings for deprecated API

**Comment by cbscd** [comment]
@robertwijas It would be really helpful if you publish the beta 2.0 branch, so you can get some help from us. :)

**Comment by ajubbal** [comment]
I agree with @cbscd if you could push up the branch others can start using and building upon what you have. :)

**Comment by cbscd** [comment]
This project seems to be abandoned, what a shame!

**Comment by ajubbal** [comment]
Ended up going with https://github.com/tombenner/nui

**Comment by cbscd** [comment]
Thanks, I'll check it out.

**Comment by robertwijas** [comment]
I'll need to clean up some stuff and remove unfinished features before releasing alpha version here.
If you want to get pre-alpha stuff please [subscribe to UISS mailing list.](http://eepurl.com/bnNaq1). Thanks!
